tests/unit/header-cell-test.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import {
+  moduleFor,
+  test
+} from 'ember-qunit';
+
+import ColumnDefinition from '../../models/column-definition';
+
+moduleFor('view:headerCell', 'Unit: Header Cell', {
+  unit: true,
+  needs: [
+    'template:header-cell'
+  ]
+});
+
+test('The elementSizeDidChange scheduled task is correctly canceled when the element is destroyed', function(assert) {
+  assert.expect(3);
+
+  const headerCell = this.subject({
+    content: ColumnDefinition.create({}),
```

Comment:
There's a bunch of initialization stuff it does that's needed, it can't just be a raw object.